src/uta/loading.py

```diff
@@ -681,6 +681,33 @@ def _fetch_origin_by_name(name):
                 cds_md5=cds_md5,
             )
             session.add(u_tx)
+
+            if ti.transl_except:
+                # if transl_except exists, it looks like this:
+                # (pos:333..335,aa:Sec);(pos:1017,aa:TERM)
```

Comment:
What interval convention is being used in the genbank file? I assume 1 based closed. Should we be converting them to 0 based half open to be consistent with all other coordinates in the database?

Reply:
Yes, good point